components/service/glean/src/test/java/mozilla/components/service/glean/GleanTest.kt

```diff
@@ -389,26 +387,31 @@ class GleanTest {
         )
     }
 
+    @Test
     fun `Make sure a11y services are collected`() {
-        var applicationContext = RuntimeEnvironment.application.applicationContext
-        var accessibilityManager = applicationContext.getSystemService(Context.ACCESSIBILITY_SERVICE) as AccessibilityManager
+        val applicationContext = ApplicationProvider.getApplicationContext<Context>()
+        val accessibilityManager = applicationContext.getSystemService(Context.ACCESSIBILITY_SERVICE) as AccessibilityManager
 
-        var shadowAccessibilityManager = shadowOf(accessibilityManager)
+        val shadowAccessibilityManager = shadowOf(accessibilityManager)
         shadowAccessibilityManager.setEnabled(true)
 
         val serviceSpy1 = spy<AccessibilityServiceInfo>(AccessibilityServiceInfo::class.java)
-        doReturn("service1").`when`(serviceSpy1).loadDescription(anyObject())
+        doReturn("service1").`when`(serviceSpy1).id
 
         val serviceSpy2 = spy<AccessibilityServiceInfo>(AccessibilityServiceInfo::class.java)
-        doReturn("service2").`when`(serviceSpy2).loadDescription(anyObject())
+        doReturn("service2").`when`(serviceSpy2).id
+
+        val nullDescription = spy<AccessibilityServiceInfo>(AccessibilityServiceInfo::class.java)
+        doReturn(null).`when`(nullDescription).id
 
-        shadowAccessibilityManager.setEnabledAccessibilityServiceList(listOf(serviceSpy1, serviceSpy2))
+        shadowAccessibilityManager.setEnabledAccessibilityServiceList(
+            listOf(serviceSpy1, serviceSpy2, nullDescription)
+        )
```

Comment:
nit: The name `nullDescription` may be misleading now that we are not using the description anymore. :)